Description: Change default of seccomp_sandbox
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=735357
Last-Update: 2015-05-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/tunables.c
===================================================================
--- trunk.orig/tunables.c
+++ trunk/tunables.c
@@ -233,7 +233,7 @@ tunables_load_defaults()
   tunable_isolate_network = 1;
   tunable_ftp_enable = 1;
   tunable_http_enable = 0;
-  tunable_seccomp_sandbox = 1;
+  tunable_seccomp_sandbox = 0;
   tunable_allow_writeable_chroot = 0;
 
   tunable_accept_timeout = 60;
